tcms/signals.py

```diff
@@ -207,21 +207,29 @@ def handle_comments_pre_delete(sender, **kwargs):
 
 
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
-    """
-    Send email updates to assignee after they've been
-    assigned a bug on bug creation.
-    """
-    if not created or instance.assignee is None:
-        return
-
+    from tcms.core.helpers.comments import get_comments
     from tcms.core.utils.mailto import mailto
 
+    comments = get_comments(instance)
+
+    recipients = set(comments.values_list("user_email", flat=True))
+    recipients.add(instance.reporter.email)
+
+    if instance.assignee is not None:
```

Comment:
This condition should be `if instance.assignee` b/c comparisons to None are redundant.
I'm surprised pylint didn't complain about it. It used to.